utils/auth/google_authorizer.go

```diff
@@ -0,0 +1,128 @@
+package auth
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/labstack/echo"
+	"github.com/satori/go.uuid"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+)
+
+const googleInfoEndpoint = "https://www.googleapis.com/oauth2/v3/userinfo"
+const uclaDomain = "g.ucla.edu"
+const tokenSub = "google oauth"
+
```

Comment:
nit: move constants out to another file as enum

Reply:
This is good since we only want the constants within this file, exporting them outside would need to be globalised for use. I would merge all into single const container though.